pipeline/src/EpiAwarePipeline.jl

```diff
@@ -15,7 +15,7 @@ using CSV, Dagger, DataFramesMeta, Dates, Distributions, DocStringExtensions, Dr
 
 # Exported pipeline types
 export AbstractEpiAwarePipeline, EpiAwarePipeline, RtwithoutRenewalPipeline,
-       RtwithoutRenewalPriorPipeline
+       RtwithoutRenewalPriorPipeline, RtwithoutRenewalTestPipeline
```

Comment:
I thought the plan was to move towards EpiAwarePipeline. Based on that idea shouldn't this be `EpiAwareTestPipeline` or `EpiAwareTodyPipeline`?

Reply:
Yes, good point!